test/multiverse/suites/activemerchant/Envfile

```diff
@@ -4,8 +4,6 @@
 
 ACTIVEMERCHANT_VERSIONS = [
   [nil, 3.0],
-  ['1.121.0', 2.5, 3.0],
-  ['1.106.0', 2.4, 3.0],
   ['1.65.0', 2.4, 2.4]
```

Comment:
This'd probably be a different ticket, but I wonder if we can take off the 2.4 constraints when we see them now since we don't have any versions less than 2.4 in the mix? I forget how exactly this works.

Reply:
I think if the max version we want it to run is 2.4, we still have to include it like `2.4, 2.4` so it sees both the min and max, since passing it just one ruby version uses it as the minimum version. I think that's the same for any version we are specifying a max ruby version for.